tests/manage/noobaa/test_create_bucket.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import pytest
+
+from tests.helpers import create_unique_resource_name
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.ocs.resources import noobaa
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
+    """
+    Returns a NooBaa resource that's connected to the S3 endpoint
+    Returns:
+        s3_res: A NooBaa resource
+    """
+    s3_res = noobaa.NooBaa()
+    return s3_res
+
+
+@pytest.fixture()
+def buckets(request, noobaa_obj):
+    """
+    Deletes all buckets that were created as part of the test
+
+    Returns:
+        Empty list of buckets
+
+    """
+    created_buckets = []
+
+    def bucket_cleanup():
+        for bucket in created_buckets:
+            noobaa_obj.s3_delete_bucket(bucket)
+
+    request.addfinalizer(bucket_cleanup)
+
+    return created_buckets
+
+
+@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+@tier1
+class TestBucketCreation:
+    """
+    Test creation of a bucket
+    """
+
+    @pytest.mark.skipif(
+        condition=config.ENV_DATA['platform'] != 'AWS',
+        reason="Tests are not running on AWS deployed cluster"
+    )
+    def test_s3_bucket_creation(self, noobaa_obj, buckets):
+        """
+        Test bucket creation using the S3 SDK
```

Comment:
There is quite a bit we can utilize from existing ceph resources - will need some minor changes there but worth it

a) https://github.com/red-hat-storage/ceph-qe-scripts/tree/master/rgw/v2/tests/s3_swift
b) https://github.com/ceph/s3-tests

Reply:
That's *very* interesting - thank you!
I shared it with the rest of the NooBaa QE team and I'm sure we'll use it :)